presto-docs/src/main/sphinx/sql/prepare.rst

```diff
@@ -0,0 +1,46 @@
+.. _prepare:
+
+=======
+PREPARE
+=======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    PREPARE statement_name FROM statement
+
+Description
+-----------
+
+Prepares a statement for execution later. Prepared statements are queries that
+are saved on a per-session basis with a given name. The statement can include
+parameters in place of literals to be filled in at execution time. Parameters
+are represented by question marks.
+
+Once a statement is prepared using the :ref:`prepare` syntax above, it can be
+executed using the :ref:`execute` statement and deleted from the session using
+the :ref:`deallocate-prepare` statement. The :ref:`describe-input` statement
+provide details on the types and origins of a prepared statement's input
+parameters.
+
+Examples
+--------
+
+Prepare a select query::
+
+    PREPARE my_select1
```

Comment:
Keep the `FROM` on the previous line since it's part of the `PREPARE` and shouldn't be mixed with the query